support-frontend/app/views/newContributions.scala.html

```diff
@@ -26,7 +26,7 @@
 @defining {
 
   <div aria-hidden="true">
-    <div class="gu-content gu-content--contribution-form">
+    <div class="gu-content gu-content--contribution-form gu-content--contribution-form--placeholder">
```

Comment:
I know that this will lead to nested CSS, but I think it's actually nice to have all the placeholder CSS in one place, inside a `.gu-content--contribution-form--placeholder{}` css definition, it keep all the placeholder CSS in one place.